core/src/main/java/contacts/core/util/FieldsWhere.kt

```diff
@@ -1,570 +0,0 @@
-@file:Suppress("FunctionName")
```

Comment:
Really good thing that I asked [about this on Reddit](https://www.reddit.com/r/Kotlin/comments/rtxfdm/comment/hr19srb/?utm_source=share&utm_medium=web2x&context=3)! You have literally reduced ~500 lines of code down to 2 lines. I have no words. I learned something from this! I don't know everything about Kotlin magic!

I have given you a gold medal for this. Thank you ❤️ 

<img width="223" alt="Screen Shot 2022-01-03 at 3 32 31 PM" src="https://user-images.githubusercontent.com/4332753/147997989-206426e5-02ac-4d6b-bd48-419ea26902e5.png">

I want to give you 50,000 points but I don't have money 😄

Reply:
That is true, sometimes seemingly random things lead us to some interesting places and situations, this time it was Reddit.

Thank you so much for your appreciation. You are too kind and wholesome.
I'm already chuffed to bits with your responses, they mean a whole lot more to me than Reddit awards. But still, thank you for taking time out and showing appreciation the way you did, be it on Reddit or on GitHub.

This was my first Open Source contribution (other than my own projects), I'm glad it worked out for both of us the way it did ❤️.